src/community/l10n_community_roles.md

```diff
@@ -1,89 +1,150 @@
-# Roles within Mozilla l10n communities
+# Introduction to localization roles and community guidelines
```

Comment:
```suggestion
# Roles and guidelines in the localization community
```

Why "Introduction"? This document is supposed to be the reference.

Reply:
This was more of a title for the first part of the doc.